spec/features/send_message_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+feature "Send message" do
+  scenario "message sends successfully", js: true do
```

Comment:
can we write a test that ensures a non-admin can't send a message to all employees? Want to make 100% sure non-admins can't spam the whole org ;)